### ISSUE TITLE
[BUG] - Print cutting off horizontal content when content is too wide

### ISSUE THREAD

**Issue by lukepint-wk** [Workiva/ixbrl-viewer#374]
### What happened?

**Issue:**
When using the new print button, the generated print view/PDF can cut content off if it is too wide.

For example, if a table has many columns, the content can overflow the printable area of a standard letter size paper. Some users accommodate this by adjusting the scale to ~70% and using Landscape layout. In some cases (shown in screenshot) even those adjustments don't allow the content to fit entirely. 

**Expected Outcome:**
- Any content that "overflows" while printing should be moved onto another page to prevent content loss while printing.

**Complications:**
- We need to calculate all of the page widths before printing and determine if any would overflow standard settings. This means the user may need an option to edit scale and layout BEFORE the system print dialog is displayed to accurately determine what content overflows and should be copied to the next page.

### Version

1.1.44

### With which browsers are you experiencing the bug?

Firefox, Chrome, Safari, Microsoft Edge

### Documents

[Overflow - Inline Viewer - FERC Form.pdf](https://github.com/Workiva/ixbrl-viewer/files/10583304/Overflow.-.Inline.Viewer.-.FERC.Form.pdf)
[ferc_overflow_tables.html.zip](https://github.com/Workiva/ixbrl-viewer/files/10583312/ferc_overflow_tables.html.zip)

### Screenshots

<img width="1456" alt="Screen Shot 2023-02-03 at 4 41 08 PM" src="https://user-images.githubusercontent.com/102554227/216724830-c18481f8-2fbd-45ae-a01e-3ed718cb3797.png">

<img width="1433" alt="Screen Shot 2023-02-03 at 4 48 42 PM" src="https://user-images.githubusercontent.com/102554227/216725629-f43d64a8-1612-4b03-9484-548e4fcdc8c5.png">

**Comment by paulwarren-wk** [comment]
I don't think this is a viewer issue, as I get the same problem if I try to print the document directly from the browser without the viewer.

This could be improved by adding some additional custom CSS to the document, but it would need to be specific to the document, and I don't think it's possible to overflow columns onto a new page.  The best that could be done would be to switch some pages to landscape and scale them down.

**Comment by lukepint-wk** [comment]
> I don't think this is a viewer issue, as I get the same problem if I try to print the document directly from the browser without the viewer.

That's a good point. I updated this to be a request instead.

One idea was to use javascript to copy the table into a new fixed-size div multiple times. Then use offsets to show different portions of the table while hiding the overflow. 

Essentially this in its basic form:
- Calculate table width as `y`
- Divide `y` by the printable width to get `x`
- Replicate `x` times with offsets (and overflow hidden)
